packages/foam-vscode/src/features/link-completion.ts

```diff
@@ -32,8 +32,10 @@ export class CompletionProvider
       .lineAt(position)
       .text.substr(0, position.character);
 
+    // Requires autocomplete only if cursorPrefix mathches `[[` that NOT ended by `]]`.
```

Comment:
`mathches`->`matches`

Reply:
in these cases @jmg-duarte I would add the suggestion in the comment so it can easily be committed to the code

```suggestion
    // Requires autocomplete only if cursorPrefix matches `[[` that NOT ended by `]]`.
```